CocoaPods/iphoneos-10.0/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CSQLite [system] {
+    header "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS10.0.sdk/usr/include/sqlite3.h"
```

Comment:
@stephencelis this also affects Carthage builds – without the modulemap, the `CSQLite` imports cannot be resolved. Even worse, they're still required *after* the framework has been built (see #492).